streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -767,38 +766,65 @@ void runOnce() {
     }
 
     private void resetInvalidOffsets(final InvalidOffsetException e) {
```

Comment:
This method is a one-liner now and is only called in a single place IIRC. Maybe better to remove the method and embed the call?

Reply:
Good catch! I'll inline it.